hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -2962,19 +2965,57 @@ public TenantInfoList listTenant() throws IOException {
       throw omEx;
     }
 
-    final List<TenantInfo> tenantInfoList = new ArrayList<>();
+    final Table<String, OmDBTenantInfo> tenantStateTable =
```

Comment:
As discussed, we do not have to read from cache here.

Reply:
Yes. Removed in 710c6f0